x-pack/plugins/ui_actions_enhanced/public/drilldowns/url_drilldown/url_drilldown_scope.ts

```diff
@@ -1,39 +0,0 @@
-/*
```

Comment:
This file was deleted and inlined into url drilldown definition,

I'd preferred to leave this as it was before for couple of reasons:

1. Lower unit test coverage now, because this isn't well unit-tested as part of definition 
2. We agreed that url drilldown will provide building blocks for custom advanced URL Drilldowns implemented by solutions, I hoped that if we keep those as separate functions, then those could be reused and we will get scope objects built by similar pattern.

Reply:
Yes, I've also thought about this too, but with current changes the functionality how the template variables are generated changed quite a bit. And having this functionality artificially in another plugin was getting in the way and those are ~100 lines of code which we don't need. I think it is good if we remove it now, and if we have a need in the future to share the way variable lists are created between different URL drilldowns we can always create a shared function of that.

Reply:
I've covered variable list and Handlebars variable objects generator functions in tests.